### DISCUSSION TITLE
part-ii/2017-126/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#191] [Question comments]
Part II 2017 Representation Theory: Paper 1, Section II, G 

https://questions.tripos.org/part-ii/2017-126/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

I got that $E$ has four 2 dimensional irreps and 2 linear irreps. 

</details>

**Comment by Kiolt** [comment]
<details>
<summary>Part (c).</summary>

We don't need to do induction, different choices of $\varepsilon,\eta$ in (b) give us all the representations we need.

</details>
